.github/workflows/testing.yml

```
@@ -7,22 +7,21 @@ jobs:
     runs-on: self-hosted
 
     steps:
+      - name: Checkout
+        uses: actions/checkout@v2
 
-    - name: Checkout
-      uses: actions/checkout@v2
+      - name: Run tests
+        id: matlab-test
+        run: |
+          TEST_RESULTS=$(/usr/local/bin/matlab -nodisplay -nosplash -nodesktop -r "addpath(genpath('.')); cd('testing/unit_tests'); runtests(struct2table(dir('*.m')).name); exit;")
+          PARSED_RESULTS=$(echo $TEST_RESULTS | awk -F'com\\.' '{ n = split($2, v, "(___+)|(===+)|(---+)"); for (i = 0; ++i <= n;) { print v[i] } }')
+          echo ::set-output name=results::$PARSED_RESULTS
```

```suggestion
          PARSED_RESULTS=$(echo "$TEST_RESULTS" | awk -F'com\\.' '{ n = split($2, v, "(___+)|(===+)|(---+)"); for (i = 0; ++i <= n;) { print v[i] } }')
          echo ::set-output name=results::"$PARSED_RESULTS"
```

Might this resolve the issue with the line breaks, as suggested [here](https://stackoverflow.com/questions/22101778/how-to-preserve-line-breaks-when-storing-command-output-to-a-variable)?

---

This should have been a "comment", not "request changes".

---

Thanks for the input - it seemed the problem is how GH Actions is dealing with some chars.